src/main/java/seedu/dietmanager/DailyFoodRecord.java

```diff
@@ -68,12 +68,13 @@ public void showDailyRecord() {
     public String showBreakfast() {
         String message = "";
         for (Food food : breakfast) {
-            message = message + food.getFoodName() + " ";
+            message = message + food.getFoodName() + ", ";
         }
         if (message.equals("")) {
             message = "Oops, you have no record for this meal";
+            return message;
         }
-        return message.trim();
+        return message.substring(0, message.length() - 2);
```

Comment:
Using a substring this way is not intuitive, must make clear what you are actually abstracting